panda.py

```diff
@@ -0,0 +1,224 @@
+    'rot': [joint2_rot.real, joint2_rot.imaginary[0], joint2_rot.imaginary[1], joint2_rot.imaginary[2]],
+    'upper': joint2_uL,
+    'val': 0,
+    'vel': joint2_maxV
+
+})
+
+#Joint3
+
+joint3_pos = joint3.GetAttribute('physics:localPos0').Get()
+joint3_rot = joint3.GetAttribute('physics:localRot1').Get()
+
+data['Joint 3'] = []
+data['Joint 3'].append({
+    'axis': joint3.GetAttribute('physics:axis').Get(),
+    'lower': joint3_lL,
+    'pos': [joint3_pos[0], joint3_pos[1], joint3_pos[2]],
+    'rot': [joint3_rot.real, joint3_rot.imaginary[0], joint3_rot.imaginary[1], joint3_rot.imaginary[2]],
+    'upper': joint3_uL,
+    'val': 0,
+    'vel': joint3_maxV
+
+})
+
+#Joint4
+
+joint4_pos = joint4.GetAttribute('physics:localPos0').Get()
+joint4_rot = joint4.GetAttribute('physics:localRot1').Get()
+
+data['Joint 4'] = []
+data['Joint 4'].append({
+    'axis': joint4.GetAttribute('physics:axis').Get(),
+    'lower': joint4_lL,
+    'pos': [joint4_pos[0], joint4_pos[1], joint4_pos[2]],
+    'rot': [joint4_rot.real, joint4_rot.imaginary[0], joint4_rot.imaginary[1], joint4_rot.imaginary[2]],
+    'upper': joint4_uL,
+    'val': 0,
+    'vel': joint4_maxV
+
+})
+
+#Joint5
+
+joint5_pos = joint5.GetAttribute('physics:localPos0').Get()
+joint5_rot = joint5.GetAttribute('physics:localRot1').Get()
+
+data['Joint 5'] = []
+data['Joint 5'].append({
+    'axis': joint5.GetAttribute('physics:axis').Get(),
+    'lower': joint5_lL,
+    'pos': [joint5_pos[0], joint5_pos[1], joint5_pos[2]],
+    'rot': [joint5_rot.real, joint5_rot.imaginary[0], joint5_rot.imaginary[1], joint5_rot.imaginary[2]],
+    'upper': joint5_uL,
+    'val': 0,
+    'vel': joint5_maxV
+
+})
+
+
+#Joint6
+
+joint6_pos = joint6.GetAttribute('physics:localPos0').Get()
+joint6_rot = joint6.GetAttribute('physics:localRot1').Get()
+
+data['Joint 6'] = []
+data['Joint 6'].append({
+    'axis': joint6.GetAttribute('physics:axis').Get(),
+    'lower': joint6_lL,
+    'pos': [joint6_pos[0], joint6_pos[1], joint6_pos[2]],
+    'rot': [joint6_rot.real, joint6_rot.imaginary[0], joint6_rot.imaginary[1], joint6_rot.imaginary[2]],
+    'upper': joint6_uL,
+    'val': 0,
+    'vel': joint6_maxV
+
+})
+
+#Joint7
+
+joint7_pos = joint7.GetAttribute('physics:localPos0').Get()
+joint7_rot = joint7.GetAttribute('physics:localRot1').Get()
+
+data['Joint 7'] = []
+data['Joint 7'].append({
+    'axis': joint7.GetAttribute('physics:axis').Get(),
+    'lower': joint7_lL,
+    'pos': [joint7_pos[0], joint7_pos[1], joint7_pos[2]],
+    'rot': [joint7_rot.real, joint7_rot.imaginary[0], joint7_rot.imaginary[1], joint7_rot.imaginary[2]],
+    'upper': joint7_uL,
+    'val': 0,
+    'vel': joint7_maxV
+
+})
+
+
+mass = link0.GetAttribute('physics:mass')
+inertia = link0.GetAttribute('physics:diagonalInertia')
+
+# print(mass.Get())
+# print(inertia.Get())
+
+# print("Position", joint1.GetAttribute('physics:localPos1').Get())
+
+with open('data.json', 'w') as outfile:
+    json.dump(data, outfile, indent =2)
```

Comment:
nice.  Add in a print statement indicating completion and the name of the file generated.  We may wish to call this panda.json for now.